proto/prehog/v1alpha/teleport.proto

```diff
@@ -818,6 +843,26 @@ enum FeatureRecommendationStatus {
   FEATURE_RECOMMENDATION_STATUS_DONE = 2;
 }
 
+// LicenseLimitEvent is emitted when access to Teleport feature
+// is denied based on license limits
+//
+// PostHost event: tp.license.limit
+message LicenseLimitEvent {
+  // PostHost property: tp.license_limit
+  LicenseLimit license_limit = 1;
```

Comment:
This is specific to device trust, correct? Let's not use general names, otherwise this will grow to accommodate every feature we have.

```suggestion
  // PostHost property: tp.device_limits
  DeviceLicenseLimits device_limits = 1;
```

Reply:
This is not meant to be specific for device trust, though device trust happens to be the first one to use it. Keeping this as is. 

also fyi, the cloud changes with prehog protos is already deployed to prod with the same name.